app/uk/gov/hmrc/exports/scheduler/jobs/PurgeAncientSubmissionsJob.scala

```diff
@@ -0,0 +1,132 @@
+import scala.collection.convert.DecorateAsScala
+import scala.concurrent.duration._
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+class PurgeAncientSubmissionsJob @Inject() (
+  val appConfig: AppConfig,
+  submissionRepository: SubmissionRepository,
+  declarationRepository: DeclarationRepository,
+  unparsedNotificationRepository: UnparsedNotificationWorkItemRepository,
+  parsedNotificationRepository: ParsedNotificationRepository,
+  transactionalOps: PurgeSubmissionsTransactionalOps
+)(implicit ec: ExecutionContext)
+    extends ScheduledJob with ExportsClient with Logging with DecorateAsScala {
+
+  val submissionCollection = db.getCollection(submissionRepository.collectionName)
+  val declarationCollection = db.getCollection(declarationRepository.collectionName)
+  val notificationCollection = db.getCollection(parsedNotificationRepository.collectionName)
+  val unparsedNotificationCollection = db.getCollection(unparsedNotificationRepository.collectionName)
+
+  override val name: String = "PurgeAncientSubmissions"
+
+  override def interval: FiniteDuration = jobConfig.interval
+  override def firstRunTime: Option[LocalTime] = Some(jobConfig.elapseTime)
+
+  private def jsonSettings(mode: JsonMode) = JsonWriterSettings.builder.outputMode(mode).build
+
+  private val jobConfig = appConfig.purgeAncientSubmissions
+  private val clock: Clock = appConfig.clock
+
+  private val latestStatus = "latestEnhancedStatus"
+  private val statusLastUpdated = "enhancedStatusLastUpdated"
+
+  private val expiryDate = Codecs.toBson(ZonedDateTime.now(clock).minusDays(180))
+
+  private val latestStatusLookup =
+    or(equal(latestStatus, "GOODS_HAVE_EXITED"), equal(latestStatus, "DECLARATION_HANDLED_EXTERNALLY"), equal(latestStatus, "CANCELLED"))
+
+  private val olderThanDate = lte(statusLastUpdated, expiryDate)
+
+  override def execute(): Future[Unit] = {
+
+    implicit val formatNotification: Format[ParsedNotification] = ParsedNotification.format
+    implicit val formatWorkItem: Format[WorkItem[UnparsedNotification]] = WorkItem.workItemRestFormat(UnparsedNotification.format)
+    import ExportsDeclaration.Mongo._
+
+    val submissions: List[Submission] = submissionCollection
+      .find(and(olderThanDate, latestStatusLookup))
+      .asScala
+      .flatMap {
+        parseJsonFromDocument[Submission]
+      }
+      .toList
+
+    logger.info(s"Submissions found older than 180 days: ${submissions.size}")
+
+    val declarations: List[ExportsDeclaration] = declarationCollection
+      .find(in("id", submissions.map(_.uuid): _*))
+      .asScala
+      .flatMap {
+        parseJsonFromDocument[ExportsDeclaration]
+      }
+      .toList
+
+    logger.info(s"Declarations found linked to submissions: ${declarations.size}")
+
+    val parsedNotifications: List[ParsedNotification] = notificationCollection
+      .find(in("actionId", submissions.flatMap(_.actions.filter(_.requestType == SubmissionRequest).map(_.id)): _*))
```

Comment:
WHat about `CancellationRequest`s? Because you'll get notifications linked separately to those too.